src/TLuaInterpreter.cpp

```diff
@@ -309,6 +309,11 @@ int TLuaInterpreter::raiseEvent( lua_State * L )
             pE.mArgumentList.append( QString(lua_tostring( L, i )) );
             pE.mArgumentTypeList.append( ARGUMENT_TYPE_STRING );
         }
+        else if( lua_isboolean( L, i ) )
```

Comment:
Ah, I think I get you, and if you cast nil to a boolean, you get false I understand.  I suspect we can get away without having a `ARGUMENT_TYPE_NIL` !  OK, I guess we are good to go with this development branch PR.  I'll leave it for Vadim to merge, and he can say whether he wants a release_30 branch PR as well - or not.